src/modules/fancyzones/editor/FancyZonesEditor/GridZone.xaml.cs

```diff
@@ -37,10 +37,10 @@ public partial class GridZone : UserControl
         public double[] HorizontalSnapPoints { get; set; }
 
         private readonly Rectangle _splitter;
-        private bool _switchOrientation = false;
+        private bool _switchOrientation;
```

Comment:
Same question as above, why not explicitly set to false?

Reply:
FxCop throws error.  It isn't needed to set when it is the default value.

Reply:
Let's not block this PR, but I have to say that I really don't like the idea because if I review some code, how can I know that the dev has intentionally not initialized to false, or just forgot to initialize it?
You see my point?